components/MenuContent.tsx

```diff
@@ -0,0 +1,31 @@
+import { Routes } from "@/app/routes/routes";
+import { useRouter } from "next/navigation";
+import '@/app/styles/menu.scss';
+
+export default function MenuContent() {
```

Comment:
This should not be MenuContent. It's more like Dashboard or ApiShortcuts or something like that. Make the file name changes for scss files also.